### PR TITLE
dev-java/spec-alpha: bumping to new version

### PR DESCRIPTION
**Pull request by scardracs**
Bumping to spec-alpha-0.2.194

Package-Manager: Portage-3.0.17, Repoman-3.0.2
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

**Comment by scardracs** [comment]
@tgbugs I need you help to fix a problem with #19991, as you are the original maintainer of spec-alpha... Maybe you can understand why it isn't working (check error provided by @juippis for more informations)

File: dev-java/spec-alpha/spec-alpha-0.2.194.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+EAPI=7
+
+inherit java-pkg-2 java-ant-2
+
+EGIT_REF="fb55378"
+
+MY_P=spec.alpha-${PV}
+
+DESCRIPTION="A Clojure library to describe the structure of data and functions."
```
Comment (juippis) [inline dev-java/spec-alpha/spec-alpha-0.2.194.ebuild:12]:
```suggestion
DESCRIPTION="A Clojure library to describe the structure of data and functions"
```

File: dev-java/spec-alpha/spec-alpha-0.2.194.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+LICENSE="EPL-1.0"
+SLOT="0.2"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+IUSE=""
+
+COMMON_DEPEND="dev-java/ant-core:0"
+RDEPEND=">=virtual/jre-1.8:*"
+DEPEND="
+	${COMMON_DEPEND}
+	>=virtual/jdk-1.8:*"
```
Comment (juippis) [inline dev-java/spec-alpha/spec-alpha-0.2.194.ebuild:25]: It doesn't look like `COMMON_DEPEND` is needed here at all.

File: dev-java/spec-alpha/spec-alpha-0.2.194.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+EANT_EXTRA_ARGS="-Dmaven.build.finalName=${MY_P}"
+
+src_prepare() {
+	default
+	cp "${FILESDIR}/build.xml" . || die
+}
+
+src_install() {
+	java-pkg_newjar "target/${MY_P}.jar"
+	dodoc CONTRIBUTING.md README.md
```
Comment (juippis) [inline dev-java/spec-alpha/spec-alpha-0.2.194.ebuild:39]: These should be matched by default `einstalldocs` call, but I don't mind either way which you call.

Comment (scardracs) [reply dev-java/spec-alpha/spec-alpha-0.2.194.ebuild:39]: So einstalldoc ... instead of dodoc ... ?

Comment (juippis) [reply dev-java/spec-alpha/spec-alpha-0.2.194.ebuild:39]: Your choice. There's a possibility that `einstalldocs` matches more doc files, but I haven't looked at the source tarball...

File: dev-java/spec-alpha/spec-alpha-0.2.194.ebuild (hunk 2 above)
Comment (scardracs) [reply dev-java/spec-alpha/spec-alpha-0.2.194.ebuild:25]: Moved dev-java/ant-core:0 to DEPEND

File: dev-java/spec-alpha/spec-alpha-0.2.194.ebuild (hunk 1 above)
Comment (scardracs) [reply dev-java/spec-alpha/spec-alpha-0.2.194.ebuild:12]: Fixed

**Comment by scardracs** [comment]
@tgbugs if you have some notes please tell me

**Comment by tgbugs** [comment]
@scardracs looks ok to me.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**